Directory: `core/` causing error in code-quality.yaml

A pull request triggers: [code-quality.yaml](https://github.com/user-attachments/files/23913483/code-quality.yaml) that throws this error:
```text
core/:0:1: E902 FileNotFoundError: [Errno 2] No such file or directory: 'core/'
Error: Process completed with exit code 1.
```
Full log: [logs_51361181074.zip](https://github.com/user-attachments/files/23913589/logs_51361181074.zip) or here: https://github.com/thomjeff/run-density/actions/runs/19902889357/job/57051377891 

It appears the following `core/` in this block is incorrect: 

```yaml
      - name: Run flake8 checks (complexity + style)
        run: |
          echo "🔍 flake8: complexity and lint checks"
          flake8 app/ core/ \
            --max-complexity=15 \
            --select=B001,C901,E,F,W \
            --exclude=tests/*
```

There is no longer a core directory -- all .py files are in /app, and core/ could be removed to allow ```code quality.yaml``` execute without errors.

--- a/.github/workflows/code-quality.yaml
+++ b/.github/workflows/code-quality.yaml
@@ -30,7 +30,7 @@ jobs:
       - name: Run flake8 checks (complexity + style)
         run: |
           echo "🔍 flake8: complexity and lint checks"
-          flake8 app/ core/ \
+          flake8 app/ \
             --max-complexity=15 \
             --select=B001,C901,E,F,W \
             --exclude=tests/*
@@ -38,9 +38,9 @@ jobs:
       - name: Check code formatting with black
         run: |
           echo "🧱 black: formatting check"
-          black app/ core/ --check
+          black app/ --check
 
       - name: Check import sorting with isort
         run: |
           echo "🔃 isort: import order check"
-          isort app/ core/ --check-only --profile black
+          isort app/ --check-only --profile black

